src/transformers/builder.ts

```diff
@@ -0,0 +1,34 @@
+import { UrlParser, UrlTransformer } from "../types.ts";
+import {
+  getNumericParam,
+  setParamIfDefined,
+  setParamIfUndefined,
+} from "../utils.ts";
+
+export const parse: UrlParser<{ fit?: string }> = (url) => {
```

Comment:
```suggestion
export const parse: UrlParser<{ quality?: number }> = (url) => {
```
I think this is the correct type